syntax = "proto3";

package web1;

service Web1 {
    rpc SayHello (HelloRequest) returns (HelloReply) {};
    rpc SayHelloAgain (HelloRequest) returns (HelloReply) {};
}

message HelloRequest {
    string name = 1;
}

message HelloReply {
    string message = 1;
}